solutions/usaco-785.mdx

```diff
@@ -2,16 +2,41 @@
 id: usaco-785
 source: USACO Bronze 2018 January
 title: Out of Place
-author: Maggie Liu
+author: Maggie Liu, Ryan Chou
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_outofplace_bronze_jan18.html)
 
+**Time Complexity:** $\mathcal{O}(N)$
+## Implementation
```

Comment:
implementation should go above time complexity

Reply:
alright, thanks!